src/containers/Dashboard/DashboardContainer.tsx

```diff
@@ -13,28 +13,40 @@ import {useSelector, useDispatch} from "react-redux";
 import {RootState} from "../../app/rootReducer";
 import {useHistory} from "react-router-dom";
 import {Routes} from "../../constants/routes";
-import {getNodeList} from "./NodeSlice";
+import {getNodeList, getDiskInfo, getGeneralInfo} from "./NodeSlice";
 
 export const DashboardContainer = (): ReactElement => {
-    const [areElementsHidden, setElementsHidden] = useState(true);
+    const [areElementsHidden, setElementsHidden] = useState<boolean>(true);
+    const [selectedNodeIndex, setSelectedNodeIndex] = useState<number>(0);
     const history = useHistory();
-    const state = useSelector((state: RootState) => state);
     const dispatch = useDispatch();
+    const state = useSelector((state: RootState) => state);
+    const token = state.user.token;
+    const nodeList = state.node.nodeList;
 
     useEffect(() => {
-        if (!state.user.token) {
+        if (!token) {
```

Comment:
Create a new Authorized container that will contain this logic (put it in App somewhere after Routes so Dashboard container will be children). That way it can be used in all future pages/containers.